src/AnalyzerConfiguration/NI.CSharp.Analyzers.globalconfig

```diff
@@ -148,6 +152,12 @@ dotnet_diagnostic.CA1506.severity = none
 
 dotnet_diagnostic.CA1507.severity = none
 
+# CA1508: Avoid dead conditional code
```

Comment:
NOTE:  This is disabled from ASW, and I think it should broadly apply to all NI code.